src/main/java/org/datadog/jmxfetch/RemoteConnection.java

```diff
@@ -20,6 +20,7 @@
     private String jmx_url;
     private static final String TRUST_STORE_PATH_KEY = "trust_store_path";
     private static final String TRUST_STORE_PASSWORD_KEY = "trust_store_password";
+    private static final String DEFAULT_RMI_RESPONSE_TIMEOUT = "5000";
```

Comment:
based on the docs this is expressed in milliseconds, are you sure 5 seconds is enough in usual cases? (I know some JVMs expose a lot of beans, and on these environments I could imagine that the call to get all the beans takes more than 5 seconds).

Reply:
I bumped this up to 15 by default and am using the user defined "refresh_beans" parameter to address this and make it configurable. What do you think?

Reply:
I bumped this again to actually be the default of the refresh_bean timeout as I think that makes the most sense here.